launcher/commands/install.py

```diff
@@ -173,6 +173,18 @@ def _create_full_install_args() -> Dict:
 
     return arguments
 
+def replace_string_in_file(file_path, target_string, replacement_string):
```

Comment:
Please use type hints in function definition:
`def replace_string_in_file(file_path: Path, target_string: str, replacement_string: str):`

Reply:
ready